components/lcd/esp_lcd_st7796/include/esp_lcd_st7796.h

```diff
@@ -40,6 +44,15 @@ typedef struct {
                                                  *   Please refer to `vendor_specific_init_default` in source file.
                                                  */
     uint16_t init_cmds_size;                    /*<! Number of commands in above array */
+#if SOC_MIPI_DSI_SUPPORTED
+    struct {
+        esp_lcd_dsi_bus_handle_t dsi_bus;               /*!< MIPI-DSI bus configuration */
+        const esp_lcd_dpi_panel_config_t *dpi_config;   /*!< MIPI-DPI panel configuration */
+    } mipi_config;
```

Comment:
question, should we use a union here in case there will a SPI/I80 specific configuration.

Reply:
I don't think a union is needed for now. It seems there are no additional configurations at the moment. If extra configurations are needed in the future, we can add them later.